Core/AppRuntime/Source/v8/LICENSE_NODE

```diff
@@ -0,0 +1,19 @@
+Copyright Node.js contributors. All rights reserved.
+

```

Comment:
is this logic distributed through a submodule/do we have to check this in?

Reply:
If we do have to check this in, we probably also need to call it out in [NOTICE.md](https://github.com/BabylonJS/BabylonNative/blob/master/NOTICE.md).